stablehlo/reference/Scope.cpp

```diff
@@ -53,10 +55,44 @@ Tensor Scope::find(Value ssaValue) const {
   return parent_->find(ssaValue);
 }
 
-SmallVector<Tensor> Scope::find(ValueRange ssaValues) const {
+SmallVector<InterpreterValue> Scope::find(ValueRange ssaValues) const {
   return llvm::to_vector(
       llvm::map_range(ssaValues, [&](Value value) { return find(value); }));
 }
 
+Tensor Scope::findTensor(Value ssaValue) const {
+  auto it = stack_frame_.find(ssaValue);
+
+  if (it != stack_frame_.end()) return it->second.getTensor();
+
+  if (!parent_)
+    llvm::report_fatal_error(llvm::formatv("value {0} not found in scope",
+                                           debugString(ssaValue).c_str()));
+
+  return parent_->find(ssaValue).getTensor();
```

Comment:
Do we need to duplicate the logic of `find`? Can we just call it, and then call `getTensor` on the result? Same below.